### DISCUSSION TITLE
Suggestions and ideas

### DISCUSSION THREAD

**Discussion by DariusIII** [NNTmux/newznab-tmux#1277] [Ideas]
## 👋 Welcome!
  Post your ideas and suggestions and we will look at them

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 08solsticegxp** [comment]
I think it would be great to have a ready to go VM that can be downloaded and fired up. Wiki documentation with it that has the username and passwords and where to change them. OR a script that can be run on bootup that asks for the passwords and usernames you want to set for things and then goes and changes them.
